python/ray/train/lightning/lightning_trainer.py

```diff
@@ -396,21 +438,12 @@ def _lightning_train_loop_per_worker(config):
         )
     trainer_config["strategy"] = RayDDPStrategy(**ddp_strategy_config)
 
-    # TODO(yunxuanx): Next PR, add logging and checkpointing support
-    trainer_config["enable_checkpointing"] = False
-
-    # Filter out existing ModelCheckpoint Callbacks
-    callbacks = []
-    for callback in trainer_config.get("callbacks", []):
-        if isinstance(callback, ModelCheckpoint):
-            logger.warning(
-                "LightningTrainer will create a Ray ModelCheckpoint callback "
-                "based on `LightningConfig.model_checkpoint_config`. All the other"
-                " ModelCheckpoint callbacks are ignored."
-            )
-        else:
-            callbacks.append(callback)
-    trainer_config["callbacks"] = callbacks
+    # AIR needs a RayModelCheckpoint for metircs logging anyway.
+    trainer_config["enable_checkpointing"] = True
+    os.environ["TUNE_DISABLE_STRICT_METRIC_CHECKING"] = "1"
```

Comment:
Hmmm, is it possible to set this with a `runtime_env` instead? Seems kind of hard though... This will propagate the env variable to future code that runs in the same script. What do other people think?